org.gabmus.hydrapaper.json

```diff
@@ -8,36 +8,13 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=wayland",
-        "--share=network",
-
         "--talk-name=org.gnome.SettingsDaemon",
-
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
-        "--filesystem=~/:rw",
-        "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
-        "--filesystem=~/.config/hydrapaper:rw",
-        "--filesystem=~/.config:rw",
-        "--filesystem=~/.cache:rw",
-        "--filesystem=~/.cache/hydrapaper:rw",
-        "--talk-name=ca.desrt.dconf",
-
-        "--env=SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt"
+        "--filesystem=~/:ro",
```

Comment:
Still not sure home access is needed? Just `xdg-pictures:ro` perhaps?

Reply:
Because one of the features is letting the user add his own wallpapers folders, that may or may not (in many cases, including mine) be inside the xdg-pictures folder

Reply:
Hmm, I suppose its fine considering the limited permissions otherwise.

You should use `--filesystem=home:ro` though.

Reply:
Changing it now

Reply:
done

Reply:
There's a problem now, using the latest manifest and trying to run hydrapaper I got the following error: https://paste.gnome.org/pmboagwfl

Reply:
should be fixed now